pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -606,3 +609,10 @@ kʲ	voiceless	velar	stop		palatalization:palatalized
 k̚	voiceless	velar	stop		release:unreleased	
 kʷʼ	voiceless	velar	stop	+	labialization:labialized,ejection:ejective	
 ɡ̤	voiced	velar	stop	+	breathiness:breathy	
+d̪l̪	voiced	lateral	affricate			
+ɽr	voiced	retroflex	trill			
+kp	voiceless	labio-velar	stop			
+gb	voiced	labio-velar	stop			
+qʡ	voiceless	uvular-epiglottal	stop			
```

Comment:
again, a case where you can say it is a cluster.

Reply:
I know these are rare sounds, but they are in IPA and the inventory sources point they are co-articulated. E.g., for Somali: http://ling.uta.edu/~jerry/somali.pdf

We can of course decide not to add double-articulation, but I vote for having them.